src/nvim/normal.c

```diff
@@ -79,7 +79,7 @@ typedef struct normal_state {
   int mapped_len;
   int old_mapped_len;
   int idx;
-  int c;
+  bool c;
```

Comment:
Here to, looking at the usage of this field afterwards, it should be kept as an int.

Reply:
Brain fart #2